src/test/kotlin/org/rust/cargo/project/RustProjectSettingsServiceTest.kt

```diff
@@ -19,11 +18,11 @@ import java.nio.file.Paths
 class RustProjectSettingsServiceTest : LightPlatformTestCase() {
 
     fun `test serialization`() {
-        val service = RustProjectSettingsServiceImpl(project)
+        val service = RustProjectSettingsService(project)
 
         @Language("XML")
         val text = """
-            <RustProjectSettings>
+            <State>
```

Comment:
nit: Do we really want to change name of this tag? I'd keep some meaningful name here.
It seems the solution is simple - rename `State` class. It also may help not to introduce several classes with the same `State` name